components/Terminal/commands/registry.ts

```diff
@@ -20,3 +20,7 @@ export function get_command(name: string) {
 export function get_help(name: string) {
   return registry[name]?.help
 }
+
+export function get_all_commands() {
```

Comment:
Do we want camelCase names or snake_case as at the moment its not consistent

Reply:
döñé